src/store/ducks/organizations.ts

```diff
@@ -0,0 +1,135 @@
+/** Organizations redux module */
```

Comment:
Should this be moved to the `opensrp` directory?

Reply:
Idk,.. maybe understanding why we have ducks that go to opensrp directory might help me make better judgements on where to place newer ducks modules.

Reply:
@p-netm its just a way to organize.  We have ducks that deal with data from Superset, and ducks that deal with data from OpenSRP.  We've currently been putting all the OpenSRP ducks in one directory.   Personally I also think that we are highly likely to move the OpenSRP ducks out to packages, or somewhere else.